### PR TITLE
N9K multisite feature

### PR DESCRIPTION
**Pull request by praveenramoorthy**
##### SUMMARY
Adding NXOS multisite support.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
nxos_bgp_af
nxos_bgp_neighbor
nxos_bgp_neighbor_af
nxos_vxlan_vtep
nxos_vxlan_vtep_vni
nxos_l3_interfaces

### CODE REVIEW

**Review by NilashishC — CHANGES_REQUESTED** [review]

File: plugins/modules/nxos_bgp_af.py [hunk 1]
```diff
@@ -213,6 +213,14 @@
     - present
     - absent
     type: str
+  retain_route_target:
```
Comment (NilashishC) [inline plugins/modules/nxos_bgp_af.py:216]: Please add a `version_added: 1.1.0` key to all the new attributes.

**Review by NilashishC — CHANGES_REQUESTED** [review]

File: plugins/modules/nxos_vxlan_vtep_vni.py [hunk 2]
```diff
@@ -84,21 +84,30 @@
     choices:
     - present
     - absent
+    type: list
```
Comment (NilashishC) [inline plugins/modules/nxos_vxlan_vtep_vni.py:87]: seems like the type of `state` key was changed to list?
```suggestion
    type: str
```

File: plugins/modules/nxos_bgp_af.py (hunk 1 above)
Comment (praveenramoorthy) [reply plugins/modules/nxos_bgp_af.py:216]: Added version

File: plugins/terminal/nxos.py [hunk 3]
```diff
@@ -64,6 +64,14 @@ class TerminalModule(TerminalBase):
             br"Cannot apply ACL to an interface that is a port-channel member",
             re.I,
         ),
+        re.compile(
```
[the remark this answers is not in the document]
Comment (praveenramoorthy) [reply plugins/terminal/nxos.py]: Made changes as suggested

**Comment by praveenramoorthy** [comment]
There was problem with rebase, will be opening a new PR with correct changes

**Pull request closed without merging**